translations/addTranslation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I don't really understand what is happening here. Maybe a comment would be helpful for the future? Or add it to the technical documentationn?

Reply:
Pure dev script. I've added description and improved usage in the file.